AsyncSwift/Views/EventDetailView.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  EventDetailView.swift
+//  AsyncSwift
+//
+//  Created by Kim Insub on 2022/09/08.
+//
+
+import SwiftUI
+
+struct EventDetailView: View {
+
+    private let event: EventModel.Event
+    @StateObject var observed = Observed()
+
+    init(event: EventModel.Event) {
+        self.event = event
+    }
+
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: 0) {
```

Comment:
```suggestion
            VStack(alignment: .leading) {
```

<img width="755" alt="image" src="https://user-images.githubusercontent.com/28520053/189539174-d1c9bda3-b052-453f-9ae9-98a884b95f24.png">


Spacing이 init default value로 nil인 듯해서 적어주지 않으면 0 으로 처리되지않을까요 ? 👀

Reply:
하지만.. 놀랍게도.. 8이 먹어있습니다.
<img width="395" alt="image" src="https://user-images.githubusercontent.com/46256034/189573303-6a3bc43a-561d-42aa-be64-8c6fdabff6c6.png">
`
//예시 뷰
VStack(alignment: .leading) {
    Text(session.title)
        .font(.system(size: 20))
        .fontWeight(.semibold)
        .padding(.vertical, 24)
        .background(.red)
    VStack(alignment: .leading, spacing: 8) {
        ForEach(session.description, id: \.self) { paragraph in
            Text(paragraph.content)
        }
    }
    .background(.red)

    .padding(.bottom, 80)
}
`

Reply:
오호 .. 공부가 되었습니다 !